azure-pipelines.yml

```diff
@@ -0,0 +1,120 @@
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+schedules:
+  - cron: 0 22 * * 0
+    branches:
+      include:
+        - develop
```

Comment:
As far as I can see, there is no develop branch on GitHub. So I'm sure how does this work

Reply:
I think this is a mistake. I will change it to master.